curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/62a8a9d876b2580943ba9351.md

```diff
@@ -1,17 +1,32 @@
 ---
 id: 62a8a9d876b2580943ba9351
-title: Step 52
+title: Step 46
 challengeType: 0
-dashedName: step-52
+dashedName: step-46
 ---
 
 # --description--
 
-You previously used an array to store strings. But arrays can store any data type. This time, your array will be storing objects. Objects are similar to arrays, but with a few differences. One difference is that objects use properties, or keys, to access and modify data.
+You previously used an array to store strings. But arrays can store any data type. This time, your array will be storing <dfn>objects</dfn>. 
 
-Objects are indicated by curly braces. An empty object would look like `{}`.
+Objects are known as non-primitive data types which means they are mutable. Mutable data types are able to be changed after they are created.
 
-Add an empty object to your `locations` array.
+Objects are similar to `arrays`, except that instead of using indexes to access and modify their data, you access the data in objects through what are called `properties`.
```

Comment:
```suggestion
Objects are similar to `arrays`, except that instead of using indexes to access and modify their data, you access the data in objects through `properties`.
```